homeassistant/components/roon/media_player.py

```diff
@@ -124,6 +122,8 @@ def __init__(self, server, player_data):
         self._attr_shuffle = False
         self._attr_media_image_url = None
         self._attr_volume_level = 0
+        self.volume_fixed = True
+        self.volume_incremental = False
```

Comment:
Make these _protected or __private:
```suggestion
        self._volume_fixed = True
        self._volume_incremental = False
```